rbac/management/group/relation_api_dual_write_group_handler.py

```diff
@@ -54,6 +56,8 @@ def __init__(
         """Initialize RelationApiDualWriteGroupHandler."""
         if not self.replication_enabled():
             return
+        if group.tenant.tenant_name == "public":
+            raise DualWriteException()
```

Comment:
Does this result in another query? Is it worth it since the workspace query will fail regardless? FWIW there is a test case that would catch if a public tenant was ever allowed through some later refactoring.

Reply:
you are right it is not worth as there is workspace query, I removed this 👍